example.gpr

```diff
@@ -11,6 +11,23 @@ project Example is
       for Default_Switches ("Ada") use ("-Og", "-g", "-gnata");
    end Compiler;
 
+   package Recordflux is
+      for Output_Dir use "generated";
+   end Recordflux;
+
+   package Prove is
+      for Proof_Switches ("Ada") use (
+         "-j0",
+         "--prover=z3,cvc5,altergo,colibri",
+         "--steps=0",
+         "--timeout=180",
+         "--memlimit=2000",
+         "--checks-as-errors",
+         "--warnings=error",
+         "--counterexamples=off"
```

Comment:
This setting is recommended for at least the automatically generated code per the [RecordFlux user guide](https://docs.adacore.com/live/wave/recordflux/html/recordflux_ug/20-overview.html#proving-and-running-the-result) as well. I believe it contributes to the speed of verification quite a bit.

In the manual code one might be more interested in the counter-examples. Maybe it would typically make sense to try to prove everything without the counterexamples initially. And only enable counterexamples when the proof fails on the manual code and the reason is not obvious otherwise.